packages/next/build/index.ts

```diff
@@ -20,7 +20,10 @@ import { PUBLIC_DIR_MIDDLEWARE_CONFLICT } from '../lib/constants'
 import { findPagesDir } from '../lib/find-pages-dir'
 import { recursiveDelete } from '../lib/recursive-delete'
 import { recursiveReadDir } from '../lib/recursive-readdir'
-import { verifyTypeScriptSetup } from '../lib/verifyTypeScriptSetup'
+import {
+  verifyTypeScriptSetup,
```

Comment:
This appears unused.

Reply:
It's being used, not by telemetry but by that file.